gestion_base_adresse/test/test_load_structure_new_db.py

```diff
@@ -37,19 +37,165 @@ def test_configure_project_with_new_db(self):
         self.assertCountEqual(expected, result)
 
     def test_calcul_num_adr(self):
-        """Test calcul numero adresse"""
+        """Test calcul numero adresse de base la voie déverrouillée pour le test à la numérotation inversée donc pair à gauche"""
+        """Suppression des points pour pouvoir tout tester"""
+        sql = ("delete from adresse.point_adresse")
+        self.cursor.execute(sql)
+
+        """Test ajout point Impair"""
```

Comment:
```suggestion
        # Test ajout point impair
```
Et idem pour les autres ci-dessous